std_msgs/Header header

int32[] labels
float32[] scores
std_msgs/Float32MultiArray boxes